Makefile

```diff
@@ -420,6 +420,16 @@ db-backup:
 db-restore:
 	./tools/backup_db/restore.sh
 
+
+# Interactive snapshot / restore
+SNAPSHOT_BINARY = ./build/snapshot
+snapshot: $(SNAPSHOT_BINARY)
+	@ $(SNAPSHOT_BINARY) snapshot
+$(SNAPSHOT_BINARY): tools/snapshot/*.go
+	cd tools/snapshot && go install && go build -o ../../build/snapshot
```

Comment:
Is `-o ../../build/snapshot` necessary if you are doing `go install`?

Reply:
Ah, good call!  Removed